src/XRP/core-xrpl-client.ts

```diff
@@ -428,7 +439,8 @@ export default class CoreXrplClient implements CoreXrplClientInterface {
     /* eslint-disable no-await-in-loop */
     while (
       latestLedgerSequence <= lastLedgerSequence &&
-      !rawTransactionStatus.isValidated
+      !rawTransactionStatus.isValidated &&
+      !this.isMalformedTransaction(rawTransactionStatus)
```

Comment:
lmk what you think of this @amiecorso - from what i can tell, this can NEVER be validated and will always past the latest ledger

Reply:
yeah! I feel like this bit of logic was nagging at me subconsciously - an early return in the case of a malformed transaction makes total sense